README.md

```diff
@@ -1,125 +1,128 @@
+- [`node/doc`](https://github.com/nodejs/node/tree/main/doc): Contains templates, styles, and frontend logic.
+- [`node/tools/doc`](https://github.com/nodejs/node/tree/main/tools/doc): Builds and validates the API docs output.
 
-## Thanks
+---
 
-- Thanks to all contributors and collaborators that make this project possible.
-- Thanks to [Chromatic](https://www.chromatic.com/) for providing the visual testing platform that helps us review UI changes and catch visual regressions.
-- Thanks to [Vercel](https://www.vercel.com/) for providing the infrastructure that serves and powers the Node.js Website
-- Thanks to [Cloudflare](https://cloudflare.com) for providing the infrastructure that serves Node.js's Website, Node.js's CDN and more.
-  - A really warm thank you to Cloudflare as we would not be able to serve our community without their immense support.
-- Thanks to [Sentry](https://sentry.io/welcome/) for providing an open source license for their error reporting, monitoring and diagnostic tools.
-- Thanks to [Crowdin](https://crowdin.com/) for providing a platform that allows us to localize the Node.js Website and collaborate with translators.
-- Thanks to [Orama](https://docs.oramasearch.com/) for providing a search platform that indexes our expansive content and provides lightning-fast results for our users.
-- Thanks to [DigitalOcean](https://www.digitalocean.com/) for generously providing Node.js with credits as part of their open source program.
+## Relevant Links
 
-[code of conduct]: https://github.com/nodejs/admin/blob/main/CODE_OF_CONDUCT.md
-[contribution guidelines]: https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md
-[content vs code]: https://github.com/nodejs/nodejs.org/blob/main/docs/content-vs-code.md
-[dependency pinning]: https://github.com/nodejs/nodejs.org/blob/main/docs/dependency-pinning.md
-[collaborator guide]: https://github.com/nodejs/nodejs.org/blob/main/docs/collaborator-guide.md
-[figma design]: https://www.figma.com/file/a10cjjw3MzvRQMPT9FP3xz
+- [Code of Conduct][]
+- [Contribution Guidelines][]
+- [Collaborator Guide][]
+- [Figma Design][]
+- [Content vs Code][]
+- [Dependency Pinning][]
+- [Translation Guidelines][]
+- [Status Page](https://status.nodejs.org/)
+
+---
+
+## Acknowledgements
+
+- Thanks to all contributors who make this project possible.
+- Thanks to [Chromatic](https://www.chromatic.com/) for visual testing.
+- Thanks to [Vercel](https://vercel.com/) for hosting infrastructure.
+- Thanks to [Cloudflare](https://www.cloudflare.com/) for CDN and web infrastructure.
+- Thanks to [Sentry](https://sentry.io/welcome/) for open source monitoring.
+- Thanks to [Crowdin](https://crowdin.com/) for localization tooling.
+- Thanks to [Orama](https://docs.oramasearch.com/) for fast, full-text search.
+- Thanks to [DigitalOcean](https://www.digitalocean.com/) for their open source credits program.
```

Comment:
I think we should keep the original wording here, please... this new wording really doesn't feel like much of a thanks, nor conveys why we're thanking them.